packages/portfolio-contract/src/pos-gmp.flows.ts

```diff
@@ -141,13 +147,28 @@ export const CCTP = {
     src: 'noble',
     dest,
   })),
-  apply: async (_ctx, amount, src, dest) => {
+  apply: async (ctx: PortfolioInstanceContext, amount, src, dest) => {
     const denomAmount: DenomAmount = { denom: 'uusdc', value: amount.value };
     const { chainId, remoteAddress } = dest;
     const destinationAddress: AccountId = `${chainId}:${remoteAddress}`;
     trace(`CCTP destinationAddress: ${destinationAddress}`);
     const { ica } = src;
+
     await ica.depositForBurn(destinationAddress, denomAmount);
+
+    // Register for confirmation and wait
+    if (ctx?.cctpClient) {
+      trace(`CCTP transaction initiated, waiting for confirmation...`);
```

Comment:
The comment only applies to the `if` case, and it's redundant with the `trace`

```suggestion
    if (ctx?.cctpClient) {
      trace(`CCTP transaction initiated, waiting for confirmation...`);
```